modules/learn/pages/data/transactions.adoc

```diff
@@ -0,0 +1,56 @@
+= Distributed ACID Transactions
+
+[abstract]
+_Distributed ACID Transactions_ guarantee that multiple documents can be updated atomically.
+
+[#transactions-and-couchbase-server]
+== Transactions and Couchbase Server
+
+_Transactions_ are operations that ensure that when multiple discrete data-elements need to be modified such that only the successful modification of all justifies the modification of any, _either_ all the modifications do occur successfully; _or_ none of them occurs.
```

Comment:
I haven't really seen 'data-elements' before.. is there a reason not to use the simpler 'documents'?

Reply:
NIT: Both 'occur' and 'occurs' are used, implying modifications is both plural and singular.

Reply:
I'm attempting to define 'transactions' at the highest level (and all the terms involved with acid), and then go down to the couchbase specifics. So is there something better that is not couchbase-specific, referring to pieces of data in different locations?

Reply:
'None' is actually singular, though people often use plural.